src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -315,13 +360,15 @@ private void generateSummaryCsv(
   // param: sliceId indicates the specific slice ID we're reporting results for (null means all)
   // param: outputPath is the path to the output file, minus its extension
   // param: extended include additional details in the output file?
```

Comment:
these and slice above it can be eliminated. (For some reason I can't select line 359 in my multi-line comment)